src/content/quickstart/openshiftsd/_index.md

```diff
@@ -121,6 +121,20 @@ Please note that the prep_for_subm.sh script has a few pre-requirements, you wil
 
 {{< subctl-install >}}
 
+### Install kubefedctl
+
+Download the kubefedctl binary and make it available on your PATH.
+
+```bash
+VERSION=0.1.0-rc3
+OS=linux
+ARCH=amd64
+curl -LO https://github.com/kubernetes-sigs/kubefed/releases/download/v${VERSION}/kubefedctl-${VERSION}-${OS}-${ARCH}.tgz
+tar -zxvf kubefedctl-*.tgz
+chmod u+x kubefedctl
+sudo mv kubefedctl /usr/local/bin/
```

Comment:
nit: Do we want to copy it to /usr/local/bin or shall we just update the PATH to include the kubefedctl folder?

Reply:
I think ~/.local/bin/ may be a better option then.

Reply:
Done.